R/simulate.r

```diff
@@ -58,5 +80,24 @@ simulate_mssql <- function() {
 
 }
 
+#' @export
+#' @rdname tbl_lazy
+simulate_hive <- function() {
+  structure(
+    list(),
+    class = c("Hive", "DBITestConnection")
```

Comment:
Oh in that case the class should contain both "DBITestConnection" and "DBIConnection" and then you can delete the `sql_select()` method